webpack.config.development.js

```diff
@@ -8,7 +8,13 @@ config.devServer = {
     ]
   },
   hot: true,
-  inline: true
+  inline: true,
+  proxy: {
+    '/api': {
+      changeOrigin: true,
+      target: 'http://localhost:3000'
```

Comment:
It'd be neat to have a way to switch to an actual site api while you're at it... maybe pass a webpack arg through?